include/Core/Metas/Modules/DeleteTermMetaModule.php

```diff
@@ -0,0 +1,174 @@
+			'cron_label' => __( 'Delete Term Meta', 'bulk-delete' ),
+		);
+	}
+
+	/**
+	 * Render the Modules.
+	 */
+	public function render() {
+		?>
+		<!-- Term Meta box start-->
+		<fieldset class="options">
+		<?php
+		$taxonomies = $this->get_taxonomies();
+		?>
+		<h4><?php _e( 'Select the taxonomy whose term meta fields you want to delete', 'bulk-delete' ); ?></h4>
+		<table class="optiontable">
+		<?php
+		foreach ( $taxonomies as $taxonomy ) {
+			?>
+			<tr>
+				<td>
+					<input name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_taxonomy" value = "<?php echo esc_html( $taxonomy ); ?>" type = "radio" class = "smbd_<?php echo esc_attr( $this->field_slug ); ?>_taxonomy">
+					<label for="smbd_<?php echo esc_attr( $this->field_slug ); ?>_taxonomy"><?php echo esc_html( $taxonomy ); ?> </label>
+				</td>
+			</tr>
+			<?php
+		}
+		?>
+		</table>
+
+		<h4><?php _e( 'Choose your term want to delete', 'bulk-delete' ); ?></h4>
+		<table class="optiontable">
+			<tr>
+				<td>
+					<select class="enhanced-terms-dropdown" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term">
+						<option><?php _e( 'Choose Terms', 'bulk-delete' ); ?></option>
+					</select>
+				</td>
+			</tr>
+		</table>
+
+		<h4><?php _e( 'Select the term meta that you want to delete', 'bulk-delete' ); ?></h4>
+		<table class="optiontable">
+			<tr>
+				<td>
+					<select class="enhanced-term-meta-dropdown" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term_meta">
+						<option><?php _e( 'Choose Term Meta', 'bulk-delete' ); ?></option>
+					</select>
+
+					<select name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term_meta_option">
+						<option value="equal"><?php _e( 'Equal to', 'bulk-delete' ); ?></option>
+						<option value="not_equal"><?php _e( 'Not equal to', 'bulk-delete' ); ?></option>
+					</select>
+
+					<input type="text" name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_term_meta_value" />
+				</td>
+			</tr>
+		</table>
+
+		<?php
+		/**
+		 * Add more fields to the delete term meta field form.
+		 * This hook can be used to add more fields to the delete term meta field form.
+		 *
+		 * @since 6.0.0
+		 */
+		do_action( 'bd_delete_term_meta_form' );
+		?>
+
+		</fieldset>
+
+		<p>
+			<button type="submit" name="bd_action" value="delete_meta_term" class="button-primary"><?php _e( 'Bulk Delete ', 'bulk-delete' ); ?>&raquo;</button>
+		</p>
+		<!-- Term Meta box end-->
+		<?php
+	}
+
+	/**
+	 * Convert user input to bulkwp standard.
+	 *
+	 * @param array $request Request array.
+	 * @param array $options User options.
+	 *
+	 * @return array User options.
+	 */
+	protected function convert_user_input_to_options( $request, $options ) {
+		$options['term'] = esc_sql( bd_array_get( $request, 'smbd_' . $this->field_slug . '_term', 'term' ) );
+
+		$options['term_meta']       = bd_array_get( $request, 'smbd_' . $this->field_slug . '_term_meta', 'term_meta' );
```

Comment:
Call `sanitize_text_field` function on the user input.

Reply:
@sudar how to use this method?